Dockerfile.IntelVSP.rhel

```diff
@@ -0,0 +1,24 @@
+FROM registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.22-openshift-4.18 AS builder
+ARG TARGETOS
+ARG TARGETARCH
+
+COPY . /usr/src/ipu-opi-plugin
+WORKDIR /usr/src/ipu-opi-plugin
+RUN CGO_ENABLED=0 GOOS=${TARGETOS:-linux} GOARCH=${TARGETARCH} go build -a -o bin/ipuplugin cmd/intelvsp/intelvsp.go
+
+FROM registry.ci.openshift.org/ocp/4.19:base-rhel9
+ENV PYTHONUNBUFFERED=1
+ENV P4_NAME=fxp-net_linux-networking
+RUN dnf install -y NetworkManager openvswitch3.4 iproute python3 python3-pip openssh-clients
+#By setting workdir, the directory is created automatically
+WORKDIR /opt/p4/p4-cp-nws/bin/
+
+COPY ./cmd/intelvsp/fxp-net_linux-networking.pkg /
+COPY ./cmd/intelvsp/p4rt-ctl /opt/p4/p4-cp-nws/bin/
+#TODO: Update to newer package, according to release.
+COPY ./cmd/intelvsp/p4runtime-2023.11.0-py3-none-any.whl /opt/p4/p4-cp-nws/bin/
+RUN python3 -m pip install --no-cache-dir /opt/p4/p4-cp-nws/bin/p4runtime-2023.11.0-py3-none-any.whl grpcio'=='1.59.3 protobuf'=='4.25.0 netaddr'=='0.9.0
```

Comment:
Is there plans to move away from python to golang? These untracked dependencies with Python may be a red flag for product security.

Reply:
This is an ongoing discussion with Intel https://issues.redhat.com/browse/IIC-442